config/passport-config.js

```diff
@@ -0,0 +1,53 @@
+const LocalStrategy = require("passport-local").Strategy;
+const bcrypt = require("bcryptjs");
+
+// Load User model
+var { conn, user, AccessToken, user_address } = require("../models/User");
+
+module.exports = function (passport) {
+  passport.use(
+    new LocalStrategy(
+      { usernameField: "username" },
+      (username, password, done) => {
+        // Match user
+        user
+          .findOne({
+            username: username,
+          })
+          .then((user) => {
+            if (!user) {
+              res.status(500).json({
+                error: 0,
+                message: "username not found",
+                data: null,
+              });
```

Comment:
as _username is not found_ means it is an error so the 
response error key should be 1.
like:
```
              res.status(500).json({
                error: 1,
                message: "username not found",
                data: null,
              });
```